rest/response.go

```diff
@@ -71,7 +72,7 @@ func (s DefaultResponseSender) Send(ctx context.Context, w http.ResponseWriter,
 // FormatItem implements ResponseFormatter.
 func (f DefaultResponseFormatter) FormatItem(ctx context.Context, headers http.Header, i *resource.Item, skipBody bool) (context.Context, interface{}) {
 	if i.ETag != "" {
-		headers.Set("Etag", `"`+i.ETag+`"`)
+		headers.Set("Etag", `W/"`+i.ETag+`"`)
```

Comment:
@rs -- for the latest idea of root-item E-tag + strong E-tag from #157 -- should we set it (as weak) hear, and then override it in the `DefaultResponseSender` to be:

```go
rootEtag := strings.StripPtefix(headers.Get("Etag"), "W/")
if rootEtag != "" {
    headers.Set("Etag", rootEtag + '-' + md5Etag)
} else {
    headers.Set("Etag", md5Etag)
}
```

Reply:
Why not. We could event only do the MD5 if we detect cross resource projections.

Reply:
>  if we detect cross resource projections.

How do we detect that on this level though:-) It looks like supporting this would require some interface changes... Maybe leave it as is for now? And see if there is a good way to do it later?

Reply:
ok

Reply:
Uploaded changes suggested. @smyrman Please review.